srcpkgs/nano/template

```diff
@@ -10,8 +10,8 @@ maintainer="Lugubris <lugubris@disroot.org>"
 license="GPL-3.0-or-later"
 homepage="https://www.nano-editor.org/"
 changelog="http://git.savannah.gnu.org/cgit/nano.git/plain/NEWS"
-distfiles="https://www.nano-editor.org/dist/v4/nano-${version}.tar.xz"
-checksum=6e3438f033a0ed07d3d74c30d0803cbda3d2366ba1601b7bbf9b16ac371f51b4
+distfiles="https://www.nano-editor.org/dist/latest/nano-${version}.tar.xz"
```

Comment:
why not /v5/ instead of /latest/?

Reply:
Good point, `latest` will probably break fetching when it points to `v6`. But https://www.nano-editor.org/dist/latest probably ought to be the `site` in an `update` file to grab major version changes in `update-check`.

Reply:
Perhaps parse the version to get the `vX` path?